src/playbooks/with_build/upgrade_before_vrs.yml

```diff
@@ -8,7 +8,7 @@
 - hosts: vsds
   gather_facts: no
   tasks:
-    - name: Check if VSD prereq satisfied for predeploy
+    - name: Check if VSD prereq satisfied for preupgrade
```

Comment:
These were correctly "predeploy".  There is a predeploy step in upgrade where the prereq need to be satisfied.  I'll fix these guys on github editor.